### DISCUSSION TITLE
Octolamp build

### DISCUSSION THREAD

**Discussion by RamblingGeekUK** [martinwoodward/octolamp#8] [Show and tell]
https://user-images.githubusercontent.com/7108949/217106683-ff917d1f-3b28-465b-8d7d-e6fef8b22ce6.mov

https://user-images.githubusercontent.com/7108949/217106693-6750daa9-49f5-4307-a4b6-70bf171109a8.mov

**Comment by martinwoodward** [comment]
Oh my!  What a fabulous build.  I'm also super jealous of your maker battlestation.  #lifegoals

**Reply by RamblingGeekUK** [reply]
Thank You.
